astropy/coordinates/earth.py

```diff
@@ -848,3 +864,84 @@ def _to_value(self, unit, equivalencies=[]):
             equivalencies = self._equivalencies
         new_array = self.unit.to(unit, array_view, equivalencies=equivalencies)
         return new_array.view(self.dtype).reshape(self.shape)
+
+
+class BaseGeodeticRepresentation(BaseRepresentation):
```

Comment:
I think this will get registered in `REPRESENTATION_CLASSES`, which is not ideal... Pinging @adrn for ideas; should we allow a `name = False` explicit way to avoid registration? (Note: this should not have to hold up this PR!)